src/Database/PostgreSQL/Simple/FromField.hs

```diff
@@ -152,6 +153,15 @@ import qualified Data.UUID.Types as UUID
 import           Data.Scientific (Scientific)
 import           GHC.Real (infinity, notANumber)
 
+#if MIN_VERSION_base(4,8,0)
+import           Data.Functor.Identity (Identity(Identity))
+#endif
```

Comment:
I'm not sure why, but this was passing CI for older GHC versions. To play it safe, I went ahead and added a guard here for base 4.8.

Reply:
This is no longer applicable since we are already depending on transformers